app/EventHandlers/AddIndicatorAttributeForJsonRequests.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App\EventHandlers;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\KernelEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Add indicator attribute `request_stack.is_json` to request for JSON/JSONP requests,
+ * based on the HTTP `Accept` (accepted content type) header.
+ *
+ * @todo When https://phabricator.wikimedia.org/T263436 is done, use Symfony's "_format" parameter instead,
+ *       see https://symfony.com/doc/current/routing.html#special-parameters
+ */
+class AddIndicatorAttributeForJsonRequests implements EventSubscriberInterface {
+
+	private const PRIORITY = 64;
```

Comment:
Where do these priority numbers come from?

Reply:
Out of thin air 😬 
Silex used 512 and -512 for its methods, but such huge numbers felt too big for me, so I chose lower exponents of 2 whenever I needed a priority.